File: leet_easy_c++/leet_001/leet_001a.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int length = nums.size();
        vector<int> sol = {0, 0};
        for(int i=1; i<length; i++){
            for(int j=0; j<i; j++){
                if(nums[i]+nums[j]==target){
                    sol[0]=j;
                    sol[1]=i;
                    return sol;
                }
            }
        }
    }
};

File: leet_easy_java/leet_001/leet_001a.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int len = nums.length;
        HashMap<Integer, Integer> hmap = new HashMap<Integer, Integer>();
        for(int i=0; i< len; i++){
            if(hmap.get(target - nums[i]) != null){
                int[] ret = {hmap.get(target - nums[i]), i};
              return ret;  
            } 
            hmap.put(nums[i], i);
        }
        int[] err = {-1,-1};
        return err;
    }
}